src/components/Navbar.vue

```diff
@@ -15,18 +15,18 @@
 			</button>
 
 			<!-- Site logo -->
-			<RouterLink class="mr-8" to="/">
+			<LLink class="mr-8" href="/">
 				<img alt="Site Logo" class="block max-lg:hidden" height="21" src="../assets/logos/libertAI.svg" width="120" />
 				<img alt="Site Logo" class="hidden h-8 max-lg:block" src="../assets/logos/libertAI.svg" />
-			</RouterLink>
+			</LLink>
 
 			<!-- Navbar buttons for large screens -->
 			<nav class="body-small flex space-x-8 text-neutral-800 max-lg:hidden">
-				<RouterLink to="/company">Company</RouterLink>
-				<RouterLink to="/earn">Earn</RouterLink>
-				<RouterLink to="/developer">Developer</RouterLink>
-				<RouterLink to="/tokenomics">Tokenomics</RouterLink>
-				<a href="/Litepaper_0624_v2.pdf" target="_blank">Litepaper</a>
+				<LLink href="/company">Company</LLink>
+				<LLink href="/earn">Earn</LLink>
+				<LLink href="/developer">Developer</LLink>
+				<LLink href="/tokenomics">Tokenomics</LLink>
+				<LLink href="/Litepaper_0624_v2.pdf" target="_blank">Litepaper</LLink>
```

Comment:
Not a fan of all of this, use `LLink` when needed but for basic `RouterLink` it just adds useless complexity when reading the code